common/OpTestHMC.py

```diff
@@ -246,24 +248,24 @@ def wait_system_state(self, exp_state=OpManagedState.OPERATING, timeout=WAITTIME
             if count > 60:
                 raise OpTestError("Time exceeded for reaching %s" % exp_state)
 
-    def is_lpar_in_managed_system(self, mg_system=None):
+    def is_lpar_in_managed_system(self, mg_system=None, lpar_name=None):
```

Comment:
@harihare Are these changes related to DLPAR ? If not, we can have them as a separate PR please